<?xml version="1.0" encoding="UTF-8"?><testrun name="TestAlarma.queNoSePuedaActivarUnaAlarmaSiHayAlMenosUnSensorDesactivado" project="PB2-PARCIAL2-2C-2022-FITTIPALDI" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="queNoSePuedaActivarUnaAlarmaSiHayAlMenosUnSensorDesactivado" classname="parcial.alarma.fittipaldi.TestAlarma" time="0.001">
    <error>java.lang.Exception: Unexpected exception, expected&lt;parcial.alarma.fittipaldi.SensorDesactivadoException&gt; but was&lt;parcial.alarma.fittipaldi.SensorDuplicadoException&gt;&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:30)&#13;
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)&#13;
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)&#13;
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)&#13;
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:93)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:756)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)&#13;
Caused by: parcial.alarma.fittipaldi.SensorDuplicadoException&#13;
	at parcial.alarma.fittipaldi.Alarma.agregarSensor(Alarma.java:45)&#13;
	at parcial.alarma.fittipaldi.TestAlarma.queNoSePuedaActivarUnaAlarmaSiHayAlMenosUnSensorDesactivado(TestAlarma.java:75)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:19)&#13;
	... 18 more&#13;
&#13;
</error>
  </testcase>
</testrun>
